prow/spyglass/lenses/junit/template.html

```diff
@@ -69,6 +73,10 @@
                             <a href="#" class="open-stdout">open stdout<i class="material-icons" style="font-size: 1em; vertical-align: middle; padding-left: 3px;">open_in_new</i></a>
                             <pre style="display: none;">{{$indTest.Output}}</pre>
                             {{end}}
+                            {{if $indTest.Error}}
+                            <a href="#" class="open-stdout">open stderr<i class="material-icons" style="font-size: 1em; vertical-align: middle; padding-left: 3px;">open_in_new</i></a>
```

Comment:
How about the an element named "open-stderr" here?

Reply:
that implies open two tabs to check the logs and correlate both  ... ideally, we should show the information interleaved, respecting the timestamps ...

Reply:
it seems is not possible right now, so maybe having `open-stdout` and `open-stderr` is more correct?

I realized @pohly  was not tagged here

Reply:
> How about the an element named "open-stderr" here?

This is a css classname that isn't visible outside of the sourcecode. I've renamed it to `open-stdout-stderr` for now, but the name is irrelevant for the functionality

Reply:
> that implies open two tabs to check the logs and correlate both ... ideally, we should show the information interleaved, respecting the timestamps ...

Both these fields are just strings and I do not think it is reasonable to make any assumptions about their content like having timestamps present. If you look at the screenshots above, you will notice that the stdout does not have them for example.

If you feel like these two should be shown together, that is IMHO something that would need to be solved in Ginkgo, not in Prow.

Reply:
Fair enough

Reply:
I agree that we should have only a single output stream and added an item for it in https://github.com/kubernetes/kubernetes/issues/109744#issuecomment-1201135372